github-orgs/kubeflow/org.yaml

```diff
@@ -1044,6 +1044,8 @@ orgs:
             description: Kubeflow Steering Committee; Defined by Kubeflow governance
             members:
             - andreyvelich
+            - franciscojavierarceo
+            - juliusvonkohout
```

Comment:
@andreyvelich for this one I think I should actually remove James and Josh but I figured I'd confirm first.

Reply:
Yeah, we don't have other options for GitHub teams.

Reply:
Sounds good, removed.